app/models/Document.ts

```diff
@@ -100,12 +103,6 @@ export default class Document extends ParanoidModel {
 
   revision: number;
 
-  @computed
-  get emoji() {
```

Comment:
Previously if you star a document then the emoji is shown in the sidebar next to the document instead of the star, that doesn't seem to be the case anymore…